code/style.css

```diff
@@ -1,7 +1,500 @@
+
+ .mark {
+   position: absolute;
+   top: 0;
+   left: 0;
+   height: 16px;
+   width: 16px;
+   background-color: #eee;
+   border: 1px solid rgba(51, 51, 51, 0.575);
+   border-radius: 50%;
+ }
+
+ /* On mouse-over, add a background color and new border color */
+
+ .radio-style:hover input~.mark {
+   background-color: #ccc;
+   border-color: teal;
+ }
+
+ /* When the radio button is checked: */
+
+ .radio-style input:checked~.mark {
+   background-color: rgba(0, 128, 128, 0.404);
+   border-color: teal;
+   box-shadow: 0px 0px 4px teal;
+ }
+
+ /* The indicator (the dot/circle - hidden when not checked) */
+
+ .mark:after {
+   content: "";
+   position: absolute;
+   display: none;
+ }
+
+ /* Show the indicator (dot/circle) when checked */
+
+ .radio-style input:checked~.mark:after {
+   display: block;
+ }
+
+ /* Style the indicator (dot/circle) */
+
+ .radio-style .mark:after {
+   top: 4px;
+   left: 4px;
+   width: 8px;
+   height: 8px;
+   border-radius: 50%;
+   background: #fff;
+ }
+
+ /****** Custom checkbox ******/
+
+ .checkbox-container {
+   display: flex;
+   flex-direction: row;
+ }
+
+ .checkbox {
+   margin-top: 38px;
+   display: block;
+   position: relative;
+   padding-left: 30px;
+   cursor: pointer;
+   font-size: 16px;
+ }
+
+ .checkbox input {
+   position: absolute;
+   opacity: 0;
+ }
+
+ /* creating a tick mark */
+
+ .checkmark {
+   position: absolute;
+   top: 0;
+   left: 0;
+   height: 16px;
+   width: 16px;
+   background-color: #fff;
+   border: 1px solid rgba(51, 51, 51, 0.575);
+ }
+
+ .checkbox:hover input~.checkmark {
+   background-color: #ccc;
+   border-color: teal;
+ }
+
+ /* glow on focus */
+
+ input:focus~.checkmark {
+   border-color: teal;
+   box-shadow: 0px 0px 4px teal;
+ }
+
+ /* creation of indicator hidde when not checked */
+
+ .checkmark:after {
+   content: "";
+   position: absolute;
+   display: none;
+ }
+
+ /* showing the checkmark when checked */
+
+ .checkbox input:checked~.checkmark:after {
+   display: block;
+ }
+
+ /* checkmark indicator */
+
+ .checkbox .checkmark:after {
+   left: 5px;
+   top: 1px;
+   width: 4px;
+   height: 8px;
+   border: 1px solid teal;
+   border-width: 0 3px 3px 0;
+   -webkit-transform: rotate(45deg);
+   -ms-transform: rotate(45deg);
+   transform: rotate(45deg);
+ }
+
+ /******* Form buttons *******/
+
+ .align-buttons {
+   display: flex;
+   flex-direction: row;
+ }
+
+ .form-button {
+   display: flex;
+   align-self: center;
+   color: rgba(51, 51, 51, 0.822);
+   max-width: 100px;
+   margin-top: 30px;
+   padding: 10px;
+   cursor: pointer;
+   border: 2px solid teal;
+   background-color: rgba(255, 255, 255, 0.555);
+ }
+
+ .form-button:hover {
+   transform: scale(1.2);
+   transition: 0.1s;
+ }
+
+ .submit-button {
+   margin-right: 20px;
+ }
+
+ .clear-button {
+   margin-right: 20px;
+ }
+
+ .clear-button:hover {
+   border: 2px solid goldenrod;
+ }
+
+ /****************** Accordion styles **************/
```

Comment:
Love the style of your accordion, well done!

Reply:
Thank you )